source/gui/__init__.py

```diff
@@ -804,22 +804,9 @@ def _appendHelpSubMenu(self, frame: MainFrame) -> None:
 
 			self.helpMenu.AppendSeparator()
 
-			# Translators: The label for the menu item to view NVDA License document.
+			# Translators: The label for the menu item to view the NVDA License.
 			item = self.helpMenu.Append(wx.ID_ANY, _("L&icense"))
-			self.Bind(
-				wx.EVT_MENU,
-				lambda evt: systemUtils._displayTextFileWorkaround(getDocFilePath("copying.txt", False)),
```

Comment:
If `_displayTextFileWorkaround` is broken, we might as well remove it. It is private API anyway.